public/_locales/zh_TW/messages.json

```diff
@@ -0,0 +1,67 @@
+{
+	"extensionDescription": {
+		"message": "視覺化您 Thunderbird 電子郵件帳號的統計資訊"
+	},
+	"popup": {
+		"message": "",
+		"account": "帳號 | 帳號",
+		"messagesInFolder": "{0} 則訊息在 {1} 個資料夾中"
+	},
+	"stats": {
+		"message": "",
+		"mailsTotal": "郵件總數",
+		"withinYears": "{0} 年內",
+		"mailsUnread": "未閱讀郵件",
+		"niceWork": "幹得好！",
+		"percentOfReceived": "已收到郵件的 {0}%",
+		"mailsReceived": "已收到郵件",
+		"percentOfTotal": "佔郵件總數的 {0}%",
+		"mailsSent": "已寄送郵件",
+		"mailsPerMonth": "每月郵件",
+		"mailsYear": "郵件/年",
+		"mailsPerDay": "每日郵件",
+		"mailsWeek": "郵件/週",
+		"loadingInProgress": "正在讀取當前帳號的所有電子郵件……",
+		"accountEmpty": "當前帳號為空，並沒有任何郵件。",
+		"charts": {
+			"years": {
+				"title": "年",
+				"description": "每年的郵件總數"
+			},
+			"months": {
+				"title": "月",
+				"description": "每月的郵件總數"
+			},
+			"daytime": {
+				"title": "天",
+				"description": "每天的郵件總數"
+			},
+			"weekday": {
+				"title": "週",
+				"description": "每週每天的郵件數"
+			},
+			"month": {
+				"title": "月",
+				"description": "每年每月的郵件數"
+			},
+			"temporalDistribution": {
+				"title": "時間分佈",
+				"description": {
+					"received": "每週工作日中，每小時收到的郵件數",
+					"sent": "每週工作日中，每小時寄送的郵件數"
+				}
+			},
+			"leader": {
+				"title": {
+					"received": "收到最多郵件來自",
+					"sent": "寄送最多郵件去往"
+				},
+				"description": {
+					"received": "每位收件人發送的電子郵件數量”,
```

Comment:
There is a wrong quotation character at the end of this line

Reply:
Yes, I found the wrong quotation character `”` and fix it already.